Documentation improvements (#6687)
tuist/tuist: docs/docs/server/introduction/accounts-and-projects.md

---
title: Accounts and projects
titleTemplate: :title | Introduction | Server | Tuist
description: Learn how to create and manage accounts and projects in Tuist.
---

# Accounts and projects

## Accounts

To use the server, you'll need an account. There are two types of accounts:

- **Personal account:** Those accounts are created automaticaly when you sign up and are identified by a handle that's obtained either from the identity provider (e.g. GitHub) or the first part of the email address.
- **Organization account:** Those accounts are manually created and are identified by a handle that's defined by the developer. Organizations allow inviting other members to collaborate on projects.

If you are familiar with [GitHub](https://github.com), the concept is similar to theirs, where you can have personal and organization accounts, and they are identified by a *handle* that's used when constructing URLs.

> [!NOTE] CLI-FIRST
> Most operations to manage accounts and projects are done through the CLI. We are working on a web interface that will make it easier to manage accounts and projects.

> You can manage the organization through the sub commands under `tuist organization`.

Reviewer: ```suggestion
You can manage the organization through the subcommands under `tuist organization`.
```

> You can manage the organization through the sub commands under `tuist organization`.

Reviewer: Should we link to the CLI `tuist organization reference?

## Projects

> Your projects, either Tuist's or raw Xcode's, need to be integrated with your account through a server-side project. Continuing with the comparison with GitHub, it's like having a local and a remote repository where you push your changes. You can use the `tuist project` to create and manage projects.

Reviewer: What do you think about referring to `server-side projects` as remote projects or remote Tuist projects?

For example, [here](https://github.com/tuist/tuist/pull/6687/files#diff-2835abcd5971693d7257fe95c3293c5f245faa597e775c40c5def4833dc415d5R9), we could say that a remote project is required instead of "integrating your project with Tuist Server". Just a thought.

Author: I am very board with this. Let's be aligned going forward in the terminology used.

> Projects are identified by a full-handle, which is the result of concatenating the organization handle and the project handle. For example, if you have an organization with the handle `tuist`, and a project with the handle `tuist`, the full-handle would be `tuist/tuist`.

Reviewer: ```suggestion
Projects are identified by a full handle, which is the result of concatenating the organization handle and the project handle. For example, if you have an organization with the handle `tuist`, and a project with the handle `tuist`, the full-handle would be `tuist/tuist`.
```

Is there any specific reason why to use `full-handle` over `full handle`? There are a couple of extra mentions below if you decide to change that.

Author: Not really. I'll change it to full handle.

The binding between the local and the remote project is done through the configuration file. If you don't have any, create it under `Tuist/Config.swift` and add the following content:

```swift

> let config = Config(fullHandle: "{account-handle}/{project-handle") // e.g. tuist/tuist

Reviewer: ```suggestion
let config = Config(fullHandle: "{account-handle}/{project-handle}") // e.g. tuist/tuist
```
